h2-jdbc/src/main/java/org/jclouds/h2/jdbc/H2JdbcApiMetadata.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.h2.jdbc;
+
+import com.google.auto.service.AutoService;
+import com.google.common.collect.ImmutableSet;
+import com.google.inject.Module;
+import org.jclouds.apis.ApiMetadata;
+import org.jclouds.blobstore.BlobStoreContext;
+import org.jclouds.h2.jdbc.config.H2JdbcBlobStoreContextModule;
+import org.jclouds.jdbc.JdbcApiMetadata;
+
+/**
+ * Implementation of {@link H2JdbcApiMetadata} for the H2 database
+ */
+@AutoService(ApiMetadata.class)
```

Comment:
In fact, this class should be removed and let the provider metadata use the `JdbcApiMetadata` directly. That's how generic apis and providers are supposed to be used. (A provider is an API + defaults + location).